bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -457,5 +457,7 @@ spec:
       name: IBM_CS_WEBHOOK_IMAGE
     - image: quay.io/opencloudio/ibm-secretshare-operator:1.13.0
       name: IBM_SECRETSHARE_OPERATOR_IMAGE
+    - image: quay.io/opencloudio/ibm-zen-operator:1.5.0
+      name: IBM_ZEN_OPERATOR_IMAGE
```

Comment:
@horis233 

I noticed that relatedImage `IBM_ZEN_OPERATOR_IMAGE` exists in `config` file and it does not include in bundle manifest.

If we add it into bundle manifest, will it cause some issue from CICD side?

Reply:
If we add it, I don't think it will cause issue here.

I didn't add the zen operator image here is because I don't want the zen operator image to be mirrored when loading cs operator CSV by `oc mirror` since zen operator image is not a necessary component of the cs operator.

Thus I only add `IBM_ZEN_OPERATOR_IMAGE` in the deployment env.

Reply:
Thank you for the explanation!

Yes, I think this is a great reason that we should not include zen-operator image in the `relatedImage` section.

I will remove it~